tools/packaging/kernel/configs/fragments/x86_64/mmu.conf

```diff
@@ -1,4 +1,5 @@
 # x86 specific mmu/memory related items
 
 # Remove the kernel mapping from the user space - security improvement.
+CONFIG_SPECULATION_MITIGATIONS=y
```

Comment:
You're already setting this as part part of the base.conf

Reply:
Errors were being thrown during the .config file generation due to dependency config options not being present:
`CONFIG_PAGE_TABLE_ISOLATION=y` in mmu.conf has a dependency on `CONFIG_SPECULATION_MITIGATIONS=y`.
`CONFIG_RETPOLINE=y` in base.conf has a dependency on `CONFIG_SPECULATION_MITIGATIONS=y`.

I thought it best to add the same dependency in both respective files. This would avoid any .config file generation issues if one of these files was not included in the .config generation for some reason. It has no effect on the end .config as all options are merged in and redundancies are removed.

Reply:
Please. drop this one from here.
The `base` config will **always** be used, not making much sense to have this one duplicated here.

Reply:
And also from the commit message.